src/dune

```diff
@@ -1,5 +1,25 @@
-(rule
-  (targets config.mlh)
-  (mode fallback)
-  (deps config/%{profile}.mlh)
-  (action (copy "%{deps}" config.mlh)))
+(* -*- tuareg -*- *)
```

Comment:
does this make it harder to make merlin happy with our source tree?

Reply:
Not sure if I understand what you mean. This will not effect merlin in any way.